integration/kubernetes/helm-chart/alluxio/templates/_helpers.tpl

```diff
@@ -257,6 +257,24 @@ resources:
   {{- end }}
 {{- end -}}
 
+{{- define "alluxio.worker.shortCircuit.volume" -}}
+  {{- if eq .Values.shortCircuit.volumeType "hostPath" }}
+        - name: alluxio-domain
+          hostPath:
+            path: {{ .Values.shortCircuit.hostPath }}
+            type: DirectoryOrCreate
+  {{- else if eq .Values.shortCircuit.volumeType "persistentVolumeClaim" }}
+        - name: alluxio-domain
+          persistentVolumeClaim:
+            claimName: "{{ .Values.shortCircuit.pvcName }}"
+  {{- else }}
```

Comment:
I think empty dir for alluxio-domain is not helpful. The reason is that empty dir is only shared in the same pod, not across different pods. So it's not visible to Fuse pod or other client pod.

Reply:
Makes sense to me. Removed emptyDir for domain socket and keeping only PVC and hostPath